File: Day-10/158A-Next Round.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int n = scanner.nextInt();
        int k = scanner.nextInt();
        int[] scores = new int[n];
        for (int i = 0; i < n; i++) {
            scores[i] = scanner.nextInt();
        }
        int threshold = scores[k - 1];
        int advancers = 0;
        for (int score : scores) {
            if (score >= threshold && score > 0) {
                advancers++;
            }
        }
        System.out.println(advancers);
        scanner.close();
    }
}

File: Day-10/158A- Next Round.cpp
#include <iostream>
#include <vector>

using namespace std;

int main() {
    int n, k;
    cin >> n >> k;
    vector<int> scores(n);
    for (int i = 0; i < n; ++i) {
        cin >> scores[i];
    }
    int threshold = scores[k - 1];
    int advancers = 0;
    for (int score : scores) {
        if (score >= threshold && score > 0) {
            advancers++;
        }
    }
    cout << advancers << endl;
    return 0;
}
